launcher/container_runner.go

```diff
@@ -389,8 +426,17 @@ func (r *ContainerRunner) fetchAndWriteToken(ctx context.Context) error {
 				r.logger.Printf("token refreshing stopped: %v", ctx.Err())
 				return
 			case <-timer.C:
-				// Refresh token.
-				duration, err := r.refreshToken(ctx)
+				var duration time.Duration
+				// Refresh token with default retry policy.
+				err := backoff.RetryNotify(
+					func() error {
+						duration, err = r.refreshToken(ctx)
+						return err
+					},
+					retry,
+					func(err error, t time.Duration) {
+						r.logger.Printf("failed to refresh attestation service token at time %v: %v", t, err)
+					})
 				if err != nil {
 					r.logger.Printf("failed to refresh attestation service token: %v", err)
```

Comment:
Should indicate will stop retrying refreshing token

Reply:
done.